samples/dashboards/demo/climate/demo.js

```diff
@@ -977,20 +976,29 @@ async function updateBoard(board, city, column, scale, newData) {
             data: cityTable.modified
                 .getRows(void 0, void 0, ['time', column])
         });
+
+        // Update range selection
+        selectionTable.setColumns(cityTable.modified.getColumns(), 0);
+        await selectionTable.setModifier();
     }
 
-    // Update range selection
-    selectionTable.setColumns(cityTable.modified.getColumns(), 0);
     const timeRangeMax = timeRangeSelector.chart.axes[0].max;
     const timeRangeMin = timeRangeSelector.chart.axes[0].min;
-    await selectionTable.setModifier(new RangeModifier({
-        modifier: 'Range',
-        ranges: [{
+    const selectionModifier = selectionTable.getModifier();
+
+    if (
+        !selectionModifier ||
```

Comment:
![image](https://github.com/highcharts/highcharts/assets/57004120/a0461a12-2428-48ee-88f6-d242604cb66d)
I get that so i presume the condition `!selectionModifier ||` isn't ideal